.circleci/config.yml

```diff
@@ -13,13 +13,9 @@ jobs: # a collection of steps
       - restore_cache:
           key: dependency-cache-{{ checksum "backend/package.json" }}-{{ checksum "frontend/package.json" }}
       - run:
-          name: install frontend
-          working_directory: frontend
-          command: npm install
-      - run:
-          name: build frontend
-          working_directory: frontend
-          command: npm run build
+          name: prepare frontend
```

Comment:
I believe it is best to run the source npm scripts directly from the frontend folder (as before); also this is not a functional change.

You could merge the 2 original steps into 1:
```
name: prepare frontend	
working_directory: frontend	
command: npm install && npm run build
```